modules/exploits/linux/http/dtale_rce_cve_2025_0655.rb

```diff
@@ -72,12 +65,20 @@ def initialize(info = {})
     )
   end
 
+  def generate_session
```

Comment:
Could we namespace this method, as currently it could be easy to confuse this with generating a metasploit-framework-specific `session`, such as Meterpreter etc. For example:
```suggestion
  def generate_dtale_jwt
```